misc/python/materialize/cli/mzconduct.py

```diff
@@ -795,30 +813,48 @@ class ChaosDockerWorkflowStep(WorkflowStep):
     def __init__(
         self,
         service: str,
-        loop: bool,
+        other_service: str,
+        duration: int,
         run_cmd: str,
         stop_cmd: str,
         running_time: int = 60,
         stopped_time: int = 10,
     ):
         self._service = service
-        self._loop = loop
+        self._other_service = other_service
+        self._duration = duration
         self._run_cmd = run_cmd
         self._stop_cmd = stop_cmd
         self._running_time = running_time
         self._stopped_time = stopped_time
 
     def run(self, comp: Composition, workflow: Workflow) -> None:
+        if (
+            self._other_service == ""
+            and self._duration < 0
+            or self._other_service != ""
+            and self._duration >= 0
+        ):
+            raise Failed(
+                f"Need to specify either 'other_service' or 'duration' to run chaos Docker workflow step"
+            )
+
```

Comment:
Could the condition here just be `if not self._other_service and not self._duration:`?

If you move this to the `init` part it will fail faster, before any steps are run. Also, this should be `BadSpec` to signify that it's 100% an error in the way the user wrote the file, not an error at runtime.